pkg/local_workflows/filter_workflow.go

```diff
@@ -0,0 +1,123 @@
+package localworkflows
+
+import (
+	"encoding/json"
+	"fmt"
+	"slices"
+	"strings"
+
+	"github.com/snyk/go-application-framework/pkg/configuration"
+	"github.com/snyk/go-application-framework/pkg/local_workflows/content_type"
+	"github.com/snyk/go-application-framework/pkg/local_workflows/local_models"
+	"github.com/snyk/go-application-framework/pkg/workflow"
+	"github.com/spf13/pflag"
+)
+
+const (
+	FilterFindingsWorkflowName = "filterfindings"
```

Comment:
```suggestion
	FilterFindingsWorkflowName = "findings.filter"
```

suggestion: consider following the pattern established in other workflows for naming `[subject].[action]`